.github/workflows/ci-repo.yml

```diff
@@ -44,15 +53,3 @@ jobs:
 
       - name: Test
         run: npm run test -- --coverage --watchAll=false
-
-      - name: Upload to codecov (client)
```

Comment:
do we drop code coverage?

Reply:
Yes mostly because it hasn't working in a long time, but also because we never really used it.

Reply:
OK but I would move it to a separate small PR for visibility.